README.md

```diff
@@ -199,3 +269,90 @@ format is CycloneDX so the `-f` isn't needed)
 ``` bash
 bomctl export <UUID> -o merged-sbom.cdx.json
 ```
+
+### SBOM Alias
+
+Working with Multiple SBOMs, it can be tricky to keep documents straight, as different formats have different expectations for naming. `bomctl`  
+typically relies on UUIDs for storing documents, however these can be taxing to use as an ID when performing operations. The `alias` command  
+gives the option to pick a more human readable name for a document for ease of use.
+
+To get more information about `alias` and its' subcommands, run:
+
+```bash
+bomctl alias --help
+```
+
+#### __Step 1: Set Alias__
+
+We're going to use the cdx SBOM fetched in the first [exercise](#step-1-fetch). If you haven't already, go back and fetch it.
```

Comment:
Use canonical format name and try to avoid "you" statements

```suggestion
This step assumes the CycloneDX SBOM from [exercise](#step-1-fetch) was previously fetched.
```